src/dsp_tools/commands/xmlupload/check_consistency_with_ontology.py

```diff
@@ -49,11 +51,11 @@ def _find_problems_in_classes_and_properties(
     )
     msg, df = problems.execute_problem_protocol()
     if df is not None:
-        ex_name = "XML_syntax_errors.xlsx"
-        df.to_excel(excel_writer=Path(onto_check_info.save_location, ex_name), sheet_name=" ", index=False)
+        csv_file = f"XML_syntax_errors_{datetime.now().strftime('%Y-%m-%d_%H%M%S')}.csv"
+        df.to_csv(path_or_buf=Path(cwd, csv_file), index=False)
         msg += (
             "\n\n---------------------------------------\n\n"
-            f"\nAn excel: '{ex_name}' was saved at '{onto_check_info.save_location}' listing the problems."
+            f"\nThe file: '{csv_file}' was saved at '{cwd}' listing the problems."
```

Comment:
The formulation "X was saved at Y listing the problems" is not so user-friendly.
Better: "full/path.ext lists the problems"